styles.css

```diff
@@ -70,36 +69,31 @@ form > p {
 
 .study-button {
   grid-column: 1 / 3;
-  background: transparent;
-  color: #FFF;
   margin-right: 60px;
 }
 
 .meditate-button {
   grid-column: 3 / 5;
   margin: 0 30px;
-  background: transparent;
-  color: #FFF;
 }
 
 .exercise-button {
   grid-column: 5 / 7;
-  background: transparent;
-  color: #FFF;
   margin-left: 60px;
 }
 
 .activity-buttons {
   border: solid 2px #FFF;
   border-radius: 5px;
-  height: 80px;
-  width: 205px;
+  height: 70px;
+  width: 150px;
   font-size: 1.15em;
   box-sizing: border-box;
   margin-top: 5px;
+  background: transparent;
+  color: #FFF;
 }
```

Comment:
That's awesome, I didn't see this refactoring change with background and color in .activity-buttons. Way to DRY it up!